src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/AddDefendantLitigationFriendCallbackHandler.java

```diff
@@ -99,8 +111,15 @@ private CallbackResponse aboutToSubmitMultiparty(CallbackParams callbackParams)
         } else {
             CaseData.CaseDataBuilder caseDataUpdated = caseData.toBuilder()
                 .businessProcess(BusinessProcess.ready(ADD_DEFENDANT_LITIGATION_FRIEND));
-            String selectedOption = Optional.of(caseData).map(CaseData::getSelectLitigationFriend)
+
+            boolean diffSolicitors = caseData.getRespondent2SameLegalRepresentative() == null
+                || caseData.getRespondent2SameLegalRepresentative() == NO;
```

Comment:
Do you think it's possible to use flow flags from stateflowengine?

FlowFlag.TWO_RESPONDENT_REPRESENTATIVES?

Reply:
That's a good idea. Will give it ago.